example_project/Cargo.toml

```diff
@@ -4,9 +4,10 @@ version = "0.1.0"
 authors = ["Lukas Lueg <lukas.lueg@gmail.com>"]
 build = "build.rs"
 edition = "2018"
+resolver = "2"
```

Comment:
What about backward compatibility here? We can just set the features explicitly for the build-time dependency without referring to the `resolver`-key